content/2017/06/09/susy-spread.rst

```diff
@@ -0,0 +1,235 @@
+public: yes
+author: miriam
+tags: [Susy, CSS, Sass, Grids]
+image:
+  - src: 'blog/2017/spread/susy-spread.jpg'
+summary: |
+  **Susy 3.0** will be released next week,
+  if all goes well,
+  and there's a lot to write about it.
+  I wanted to start with a detailed overview
+  of one core concept: **spread**.
+
+
+Understanding ``Spread`` in Susy3
+=================================
+
+Spread isn't new to Susy3,
+`or even to Susy generally`_,
+but its full power is usually hidden behind
+other settings (like ``gutter-position``),
+or opinionated assumptions about your grid.
+In Susy3 we've gone the other direction,
+making it central to the API,
+and giving you full control over the math.
+
+.. _or even to Susy generally: http://susydocs.oddbird.net/en/latest/settings/#spread
+
+
+What is spread?
+---------------
+
+We didn't invent the idea,
+but we did come up with the term,
+because I haven't seen anyone else talking about it.
+
+
+On Containers
+~~~~~~~~~~~~~
+
+Container-spread describes how a grid-parent
+handles available gutters:
+
+Most grids only put gutters between the columns.
+That means there will be one less gutter than there are columns.
+We call that a ``narrow`` container-spread, and make it the default.
+
+.. image:: /static/images/blog/2017/spread/context-narrow.jpg
+   :alt: container-spread: narrow
+   :class: extend-small
+
+Some grids use "split" gutters,
+with half of a gutter on either size of a column –
+forming full-gutters between columns,
+and an extra half at the edges.
+Mathematically, that means we have an equal number
+of gutters and columns.
+We call that a ``wide`` spread.
+
+.. image:: /static/images/blog/2017/spread/context-wide.jpg
+   :alt: container-spread: wide
+   :class: extend-small
+
+Occasionally, a grid will have full gutters on both sides,
+meaning there is one more gutter than columns.
+We call that a ``wider`` spread.
+
+.. image:: /static/images/blog/2017/spread/context-wider.jpg
+   :alt: container-spread: wider
+   :class: extend-small
+
+
+On Spans
+~~~~~~~~
+
+Spread describes the same concept
+as it relates to internal grid-spanning elements.
+In most systems,
+including the new CSS Grids,
```

Comment:
should this say `including the new CSS Grid module`? And should we link it either to our article or to the w3c spec?